### BIG differences, this is diff of wrong files. CLUS
### "Function RATIONAL, RATIONALIZE"
### and CLHS topic selection http://clhs.lisp.se/Body/a_ration.htm
### But should be the same in contenct as "cl:functions:rationalize.diff"
### which diff two right files
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:rational.txt	2017-04-27 21:50:30.000000000 +0100
+++ saved-pages-clus/cl:functions:rational.txt	2017-04-27 21:50:30.000000000 +0100
@@ -1,4 +1,40 @@
-RATIONAL
-Please select which reference to RATIONAL you intended
+Function RATIONAL, RATIONALIZE

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Syntax
+ rational  number → rational rationalize number → rational
+Arguments and Values
+ number - a real.
+ rational - a rational.
+Description
+rational and rationalize convert reals to rationals.
+If number is already rational, it is returned.
+If number is a float, rational returns a rational that is mathematically equal in value to the float. rationalize returns a rational that approximates the float to the accuracy of the underlying floating-point representation.
+rational assumes that the float is completely accurate.
+rationalize assumes that the float is accurate only to the precision of the floating-point representation.
+Examples
+(rational 0)
+→
+0
+(rationalize -11/100)
+→
+-11/100
+(rational .1)
+→
+13421773/134217728 ;implementation-dependent
+(rationalize .1)
+→
+1/10
+Side Effects
+None.
+Affected By
+The implementation.
+Exceptional Situations
+Should signal an error of type type-error if number is not a real. Might signal arithmetic-error.
+See Also
+None.
+Notes
+Rationalizing a float by either method and then converting it back to a float of the same format produces the original number.
+(float (rational x) x)
+  ≡ (float (rationalize x) x)
+  ≡ x
+\
\ No newline at end of file
